core/kernel/persistence/smoothsql/search/ComplexSearchService.php

```diff
@@ -211,12 +211,6 @@ public function getQuery(core_kernel_persistence_smoothsql_SmoothModel $model, $
             foreach ($nextValue as $val) {
                 $criteria->addOr($this->parseValue($val));
             }
-            if($and === false) {
```

Comment:
@chrisphpgarcia but this condition added twice:
https://github.com/oat-sa/generis/blob/develop/core/kernel/persistence/smoothsql/search/ComplexSearchService.php#L193-L194
and
https://github.com/oat-sa/generis/blob/develop/core/kernel/persistence/smoothsql/search/ComplexSearchService.php#L216-L217